CondFormats/HcalObjects/interface/HcalMahiConditionsSoA.h

```diff
@@ -0,0 +1,70 @@
+                    SOA_COLUMN(HcalPedestalArray, gains_value),
+                    SOA_COLUMN(HcalPedestalArray, convertedPedestals),
+                    SOA_COLUMN(HcalPedestalArray, convertedPedestalWidths),
+                    SOA_COLUMN(float, lutCorrs_values),
+                    SOA_COLUMN(float, respCorrs_values),
+                    SOA_COLUMN(float, timeCorrs_values),
+                    // Use EIGEN_COLUMN for matrix?
+                    SOA_COLUMN(float, pedestalWidths_sigma00),
+                    SOA_COLUMN(float, pedestalWidths_sigma01),
+                    SOA_COLUMN(float, pedestalWidths_sigma02),
+                    SOA_COLUMN(float, pedestalWidths_sigma03),
+                    SOA_COLUMN(float, pedestalWidths_sigma10),
+                    SOA_COLUMN(float, pedestalWidths_sigma11),
+                    SOA_COLUMN(float, pedestalWidths_sigma12),
+                    SOA_COLUMN(float, pedestalWidths_sigma13),
+                    SOA_COLUMN(float, pedestalWidths_sigma20),
+                    SOA_COLUMN(float, pedestalWidths_sigma21),
+                    SOA_COLUMN(float, pedestalWidths_sigma22),
+                    SOA_COLUMN(float, pedestalWidths_sigma23),
+                    SOA_COLUMN(float, pedestalWidths_sigma30),
+                    SOA_COLUMN(float, pedestalWidths_sigma31),
+                    SOA_COLUMN(float, pedestalWidths_sigma32),
+                    SOA_COLUMN(float, pedestalWidths_sigma33),
+                    SOA_COLUMN(float, gainWidths_value0),
+                    SOA_COLUMN(float, gainWidths_value1),
+                    SOA_COLUMN(float, gainWidths_value2),
+                    SOA_COLUMN(float, gainWidths_value3),
+                    SOA_COLUMN(uint32_t, channelQuality_status),
+                    SOA_COLUMN(HcalQIECodersArray, qieCoders_offsets),
+                    SOA_COLUMN(HcalQIECodersArray, qieCoders_slopes),
+                    SOA_COLUMN(int, qieTypes_values),
+                    SOA_COLUMN(int, sipmPar_type),
+                    SOA_COLUMN(int, sipmPar_auxi1),
+                    SOA_COLUMN(float, sipmPar_fcByPE),
+                    SOA_COLUMN(float, sipmPar_darkCurrent),
+                    SOA_COLUMN(float, sipmPar_auxi2),
+                    SOA_SCALAR(int, maxDepthHB),
+                    SOA_SCALAR(int, maxDepthHE),
+                    SOA_SCALAR(int, maxPhiHE),
+                    SOA_SCALAR(int, firstHBRing),
+                    SOA_SCALAR(int, lastHBRing),
+                    SOA_SCALAR(int, firstHERing),
+                    SOA_SCALAR(int, lastHERing),
+                    SOA_SCALAR(int, nEtaHB),
+                    SOA_SCALAR(int, nEtaHE),
+                    SOA_SCALAR(uint32_t, offsetForHashes))
+using HcalMahiConditionsSoA = HcalMahiConditionsSoALayout<>;
+
+using HcalPSfunctorArray = std::array<float, hcal::constants::maxPSshapeBin>;  // 256
+using HcalPSfunctorBXarray = std::array<float, hcal::constants::nsPerBX>;      // 25
+
```

Comment:
These are redefined in `CondFormats/HcalObjects/interface/HcalRecoParamWithPulseShapeSoA.h`, where they are used.
Are they needed here ?

Reply:
```suggestion
```

Reply:
Removed in 4615e8825352834ac1dd4f1bbe766b0cafc1d321